src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJob.java

```diff
@@ -145,7 +183,7 @@ public Double getDurationSeconds() {
   /**
    * Sets the audio duration.
    *
-   * @param durationSeconds An Integer value to set as audio duration.
+   * @param durationSeconds A Double value to set as audio duration.
```

Comment:
I don't think it is necessary to document type in the JavaDoc.

Reply:
We can avoid this type of change if we don't replicate the typing in the javadoc.

Reply:
Was following the established patterns that we had for our other params